### PR TITLE
relationships to nicknames fix in fixtures

### PR DESCRIPTION
**Pull request by CEsGutierrez**
This corrects how fixtures call each other, using nicknames rather than the ID# (which are reassigned by rails).

### CODE REVIEW

File: test/models/category_test.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
   describe 'relations' do
     it 'has_many products' do
       expect(Category.new).must_respond_to :products
+      binding.pry
```
Comment (mfunkemomo) [inline test/models/category_test.rb]: Probably not best to keep this here for the master

**Review by mfunkemomo — APPROVED** [review]

**Review by mfunkemomo — APPROVED** [review]

**Review by calopter — APPROVED** [review]
lgtm

File: test/fixtures/order_items.yml [hunk 2]
```diff
@@ -1,119 +1,119 @@
 o_i_1:
   quantity: 60
   shipping_status: shipped 
-  product_id: 1
-  order_id: 2
+  product_id: p_1
+  product_id: o_2
```
Comment (calopter) [inline test/fixtures/order_items.yml]: `product: p_1
order: o_2`

File: test/fixtures/order_items.yml [hunk 3]
```diff
@@ -1,119 +1,119 @@
 o_i_1:
   quantity: 60
   shipping_status: shipped 
-  product_id: 1
-  order_id: 2
+  product_id: p_1
```
Comment (calopter) [inline test/fixtures/order_items.yml:4]: `product: p_1`

File: test/fixtures/order_items.yml (hunk 2 above)
Comment (calopter) [inline test/fixtures/order_items.yml:5]: `order: o_2`

**Pull request merged**